### ISSUE TITLE
CommonJS Support Does Not Work with Webpack

### ISSUE THREAD

**Issue by dlandis** [ExactTarget/fuelux#1498]
Hello,

I use webpack and I wanted to try out the repeater, so I did something like:
```
var $ = require("jquery/dist/jquery");
require("fuelux/dist/css/fuelux.css");
require("fuelux/js/repeater.js");
```

However, the error from webpack is:
```
ERROR in ./~/fuelux/js/repeater.js
Module not found: Error: Cannot resolve module 'fuelux/combobox' in C:\workspace\app\node_modules\fuelux\js
 @ ./~/fuelux/js/repeater.js 17:2-112
```

I can see in repeater.js, the code like this:

    define(['jquery', 'fuelux/combobox', 'fuelux/infinite-scroll', 'fuelux/search', 'fuelux/selectlist'], factory);

In order to work with webpack, I believe it could be like below. **Why does it need those `fuelex` prefixes**? Doesn't seem right.

    define(['jquery', './combobox', './infinite-scroll', './search', './selectlist'], factory);

Alternatively, you could change your wrapper to have the `module.exports` clause first:

	if (typeof exports === 'object') {
		// Node/CommonJS
		module.exports = factory(require('jquery'));
	} else if (typeof define === 'function' && define.amd) {
        }

I have seen some projects do it that way.

For more information about how webpack handles this situation, see [this issue](https://github.com/webpack/webpack/issues/919).

**Comment by gdborton** [comment]
@dlandis I agree that they should be using relative paths, but this library was built with requirejs's amd in mind, where typically you'd define a path for fuelux in your config.  I'm sure a pull request here would be welcome.

https://github.com/exacttarget/fuelux/blob/master/DETAILS.md#amd-support

The equivalent approach in webpack would be to use an alias.

```javascript
resolve: {
  alias: {
    fuelux: __dirname + '<path to deps>/fuelux/js' // This would be different depending on your package manager.
  }
}
```

**Note:** You'll have some additional trouble including individual modules, as the modules don't declare any of their bootstrap dependencies either.
